scss/forms/_input-group.scss

```diff
@@ -34,11 +34,21 @@ $input-prefix-padding: 1rem !default;
     }
 
     > :first-child {
-      border-radius: if($global-text-direction == rtl, 0 $input-radius $input-radius 0, $input-radius 0 0 $input-radius);
+      &:not(.input-group-button){
+        border-radius: if($global-text-direction == rtl, 0 $input-radius $input-radius 0, $input-radius 0 0 $input-radius);
+      }
+
+      &.input-group-button > * {
```

Comment:
Selectors can be merged here are the code in the same. So:
```scss
&:not(.input-group-button), &.input-group-button > * {
   ...
}
```
(don't do that, see the comment above).

Otherwise we're just duplicating code.

Reply:
I've DRY'd this. :)